### PR TITLE
Announcing v1.0.0.beta1

### PR DESCRIPTION
**Pull request by jodosha**

### CODE REVIEW

File: source/guides/projects/logging.md [hunk 1]
```diff
@@ -4,78 +4,47 @@ title: Guides - Logging
...
-      # Logger format.
-      logger.format :json
-    end
+    # ...
   end
 end
 ```
 
-Using standard output is a [best practice](http://12factor.net/logs) that most hosting SaaS companies [suggest using](https://devcenter.heroku.com/articles/rails4#logging-and-assets).
+By default it uses stardard output because it's a [best practice](http://12factor.net/logs) that most hosting SaaS companies [suggest using](https://devcenter.heroku.com/articles/rails4#logging-and-assets).
```
Comment (cllns) [inline source/guides/projects/logging.md]: `stardard` => `standard`

**Comment by jodosha** [comment]
@cllns @oana-sipos please review 😄

File: source/blog/2017-02-14-announcing-hanami-100beta1.html.markdown [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+image: true
+excerpt: >
+  Feature freeze, project logger, automatic logging of requests, SQL queries, and migrations. Minor bug fixes.
+---
+
+This `v1.0.0.beta1` release introduces Hanami [feature freeze](https://en.wikipedia.org/wiki/Freeze_(software_engineering)) for 1.0, alongside with a couple of features, and a few bug fixes.
+
+Since now on, **Hanami API's are stable and won't be changed until 2.0**.
+
+The stable release (`v1.0.0`) will happen beteen the end of March and the beginning of April 2017, which coincides with the [Hanami season in Japan](http://www.japan-guide.com/sakura/). 🌸
```
Comment (oana-sipos) [inline source/blog/2017-02-14-announcing-hanami-100beta1.html.markdown]: between instead of _beteen_

**Comment by jodosha** [comment]
@oana-sipos @cllns if you spot other typos, please push the fix 😉

**Pull request merged**